chore: create bug/feature/PR template, configure template chooser (#397)
actions/stale: .github/pull_request_template.md

## Changes

> <!-- List the change(s) you're making with this PR. -->

Reviewer: ```suggestion
<!-- List the change(s) you're making with this PR. -->
**Changes:**
- [x] ...
```

**Context**

> <!-- Explain why you're making the change(s). If you're closing an issue with this PR use the Closes keyword and the issue number like this: Closes #123 -->

Reviewer: ```suggestion
<!-- Explain why you're making the change(s). -->
<!-- If you're closing an issue with this PR, [link them with a keyword](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword). -->
```

<!-- List the change(s) you're making with this PR. -->
**Changes:**
- [x] ...

> **Context**

Reviewer: ```suggestion
## Context
```
You can use an h2 it's better sorry
